mmdet/datasets/loader/build_loader.py

```diff
@@ -61,6 +62,15 @@ def build_dataloader(dataset,
         batch_size = num_gpus * imgs_per_gpu
         num_workers = num_gpus * workers_per_gpu
 
+    def worker_init_fn(worker_id):
+        # When enable distributed training, the seed of each worker equals to
+        # num_worker * rank + worker_id + user_seed
+        # When non-distributed training, rank is 0
+        rank = torch.distributed.get_rank() if dist else 0
```

Comment:
`rank, world_size = get_dist_info()` can be moved outside, thus we can reuse the rank.